employee.rb

```diff
@@ -0,0 +1,30 @@
+class Employee
+  attr_accessor :name, :salary, :reviews, :satisfactory
+  def initialize(name, salary)
+    @name = name
+    @salary = salary
+    @reviews = []
+    @satisfactory = []
```

Comment:
Thank you for the info! By the way, sorry if I did something wrong today. I
didn't understand exactly what you meant earlier after the mini lecture.
See you tomorrow and have a goodnight!

Parker
On Jan 28, 2015 9:20 PM, "Mason F. Matthews" <notifications@github.com>
wrote:

> In employee.rb
> <https://github.com/tiyd-rails-2015-01/employee_reviews/pull/14#discussion_r23742538>
> :
>
> > @@ -0,0 +1,30 @@
> > +class Employee
> > +  attr_accessor :name, :salary, :reviews, :satisfactory
> > +  def initialize(name, salary)
> > +    @name = name
> > +    @salary = salary
> > +    @reviews = []
> > +    @satisfactory = []
>
> You set satisfactory to an array here, but you use it as a true/false
> elsewhere. It would probably be best to leave off this line to not confuse
> future developers (including yourself).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tiyd-rails-2015-01/employee_reviews/pull/14/files#r23742538>
> .
>